src/EtherFiAdmin.sol

```diff
@@ -230,22 +224,18 @@ contract EtherFiAdmin is Initializable, OwnableUpgradeable, UUPSUpgradeable {
             uint256 end = (i + 1) * 25 > _validatorsToApprove.length ? _validatorsToApprove.length : (i + 1) * 25;
 
             uint256[] memory validatorsToApproveBatch = new uint256[](end - start);
-            bytes[] memory pubKeyBatch = new bytes[](end - start);
-            bytes[] memory signatureBatch = new bytes[](end - start);
             for (uint256 j = start; j < end; j++) {
                 validatorsToApproveBatch[j - start] = _validatorsToApprove[j];
-                pubKeyBatch[j - start] = _pubKey[j];
-                signatureBatch[j - start] = _signature[j];
             }
-
-            _enqueueAdminTask("validatorsToApprove", keccak256(abi.encode(_reportHash, validatorsToApproveBatch, pubKeyBatch, signatureBatch)));
+            ValidatorApprovalTaskStatus[keccak256(abi.encode(_reportHash, _validatorsToApprove))].exists = true;
+            emit ValidatorApprovalTaskCreated(_reportHash, validatorsToApproveBatch);
         }
     }
 
-    function _handleValidators(bytes32 _reportHash, IEtherFiOracle.OracleReport calldata _report, bytes[] calldata _pubKey, bytes[] calldata _signature) internal {
+    function _handleValidators(bytes32 _reportHash, IEtherFiOracle.OracleReport calldata _report) internal {
         // validatorsToApprove
         if (_report.validatorsToApprove.length > 0) {
-            _enqueueAdminTask_ApproveValidators(_reportHash, _report.validatorsToApprove,  _pubKey, _signature);
+            _batchAdminTask_ApproveValidators(_reportHash, _report.validatorsToApprove);
```

Comment:
isn't it better to keep the naming to imply that the tasks are "enqueued", but not "executed"